MantidQt/API/test/QwtWorkspaceBinDataTest.h

```diff
@@ -19,10 +19,10 @@ class QwtWorkspaceBinDataTest : public CxxTest::TestSuite {
     for (size_t i = 0; i < 3; i++) {
       ax1->setValue(i, 10.0 + static_cast<double>(i));
       for (size_t j = 0; j < 5; j++)
-        ws->dataX(i)[j] = double(i) + double(j);
+        ws->mutableX(i)[j] = double(i) + double(j);
```

Comment:
`HistogramData::LinearGenerator`? Shouldn't use non-const access in a tight loop (I know `i` only goes to 3 but it is better not to practice this things)